truss/cli.py

```diff
@@ -297,43 +295,24 @@ def train(target_directory: str, build_dir, tag, var: List[str], vars_yaml_file,
 
 @cli_group.command()
 @click.argument("target_directory", required=False, default=os.getcwd())
-@click.option("--api-key", type=str, required=False, help="Your API key")
+@click.option("--service-name", type=str, required=True, help="Name of the service")
```

Comment:
nit: let's make this `--remote` instead of service-name

Reply:
Can we change the other `service-name` references to the remote provider in this PR for consistency?

Reply:
@joostinyi should be done already

Reply:
Meant in the remote factory code as well – I think all the `service` references would be better named `service-provider` or `remote-provider` to not overload our usage of `service` when we or other people look at this example to build new remotes off of